packages/@aws-cdk/aws-appsync/lib/data-source.ts

```diff
@@ -211,6 +240,7 @@ export interface HttpDataSourceProps extends BaseDataSourceProps {
    * The http endpoint
    */
   readonly endpoint: string;
+  readonly authorizationConfig?: HttpDataSourceAuthorizationConfig; 
```

Comment:
```suggestion
  readonly authorizationConfig?: AwsIamConfig; 
```

Reply:
remove layer of abstraction and let CDK handle the heavy lifting :) repeat elsewhere